app/src/main/java/org/apache/fineract/ui/online/loanaccounts/plannedpayment/PlannedPaymentPresenter.java

```diff
@@ -56,7 +56,7 @@ public void detachView() {
     @Override
     public void fetchPlannedPayment(String productIdentifier, String caseIdentifier,
             Integer pageIndex, String initialDisbursalDate, Boolean loadmore) {
-        this.loadmore = loadmore;
+//        this.loadmore = loadmore;
```

Comment:
I won't be able to explain you directly over here😅. It took me also quite a long time to find the flaw in the code. Briefly, I can explain in a way that in this method fetchPlannedPayment , we were first trying to set the boolean for the loadmore variable and then were invoking the fetchPlannedPayment function , but this was where the issue was as in the later function, loadmore is already being set and therefore , at the first time when we launch the plannedpayment Fragment, two times the fetchplannedpayment method is invoked , which leads to loading the data twice in the recyclerView Adapter but then when we refresh it , everything gets sorted. 
Hope I was able to make it clear :)

Reply:
Actually, I saw the code and I think loadmore is an important variable because there is another overloaded function fetchPlannedPayment which isn't taking loadmore as a parameter. So, I think what you need to do is go to [this line](https://github.com/EGOR-IND/fineract-cn-mobile/blob/592099e802aee546a145f8c9e96a3aecd149312b/app/src/main/java/org/apache/fineract/ui/online/loanaccounts/plannedpayment/PlannedPaymentFragment.java#L144) and change the passed true to false which will stop the fetchPlannedPayment method to load it twice:).

Reply:
Actually I got your point and have already tried that approach also, but the solution you are suggesting will always pass false as a parameter to the fetchPlannedPayment method, which did not seem right to me as it will then always manipulate [these](https://github.com/apache/fineract-cn-mobile/blob/592099e802aee546a145f8c9e96a3aecd149312b/app/src/main/java/org/apache/fineract/ui/online/loanaccounts/plannedpayment/PlannedPaymentPresenter.java#L79) conditional statements in the same way.

Reply:
@therajanmaurya  @jawidMuhammadi Please review and suggest which approach would be better ?  :)

Reply:
I get what you're saying, actually both of our approach will make those conditional statements give same result all the time:). So that's why I'm saying, why remove the existence of loadmore if we can pass false to it. But still I exactly unable to understand the use of loadmore as a parameter😂.

Reply:
Yes me too, let @therajanmaurya @jawidMuhammadi confirm then , I'll make the required changes .